tests/common.py

```diff
@@ -18,14 +18,36 @@
 
 import subprocess
 
+import six
+
+
+class PackageInstallationError(Exception):
+    def __init__(self, stderr="", stdout=""):
+        # type: (six.text_type, six.text_type) -> None
+        super(PackageInstallationError, self).__init__()
+        self.stderr = stderr
+        self.stdout = stdout
+
 
 def _install_rpm(file_path, upgrade=False):
-    subprocess.check_call(
+    process = subprocess.Popen(
         "rpm -{0} {1}".format("U" if upgrade else "i", file_path),
         shell=True,
         stdin=subprocess.PIPE,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
     )
 
+    process.wait()
+
+    stdout = process.stdout.read().decode("utf-8")
+    stderr = process.stderr.read().decode("utf-8")
+
+    if process.returncode != 0:
```

Comment:
Yeah, that's a good change, I was thinking about making a change like that myself :+1:

Reply:
I see it's using in multiple places so we should probably eventually refactor it in a utility function - e.g. run_command or similar.